subgraphs/livepeer/src/sdk/protocols/generic/account.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Account as AccountSchema,
+  ActiveAccount,
+} from "../../../../generated/schema";
+import { TokenManager } from "./tokens";
+import { ProtocolManager } from "./protocol";
+import { Address } from "@graphprotocol/graph-ts";
+import { CustomEventType, getUnixDays, getUnixHours } from "../../util/events";
+
+/**
+ * This file contains the AccountClass, which does
+ * the operations on the Account entity. This includes:
+ *  - Creating a new Account
+ *  - Updating an existing Account
+ *
+ * Schema Version:  2.1.0
```

Comment:
Update this line. See comments on the SDK upgrade PR

Reply:
Sure!